doc/compliancetest.rst

```diff
@@ -71,8 +50,10 @@ settings of your PC. In the properties for the driver, go to the "Advanced"
 tab. For the "Packet priority & VLAN" select "Packet priority & VLAN Disable".
 More details are given in the "Product Documentation" document for the tool.
 
-You might also need to turn off LLDP protocol for the selected network
-interface. Both Windows and Simatic TIA can have LLDP implemented.
+You also need to turn off LLDP protocol for the selected network interface,
```

Comment:
Clarify that this applies for test pc/station.

Reply:
Done